docs/content/faq/developers.mdx

```diff
@@ -170,24 +170,21 @@ Currently NBA Top Shot runs on Mainnet and NBA Top Shot test app runs on Flow Te
 
 Starting on October 16, 2021 at 7:30am PT (2:30pm UTC), there will be a flat fee of 0.00001 FLOW applied to every transaction submitted to the network. Transaction Fees are subject to change. In the event that a change does occur, the community will be informed ahead of implementation. These fees are for spam prevention and will be low and fixed for all transactions.
 
-## When will devs be able to deploy on mainnet? How can I create my first account for mainnet?
-
-During the beta period, you will need to submit your contract to the Flow team for review and then you’ll be able to submit it to the network. As soon as you’ve tested your dapp on devnet and the Flow team has reviewed your contracts, you can deploy!
+## How can I deploy a contract?
 
-Info here: [https://docs.onflow.org/dapp-deployment](https://docs.onflow.org/dapp-deployment)
+To deploy a contract, please follow the steps outlined in the [Dapp Deployment guide](/dapp-development/deployment/).
 
-## How can I deploy a contract?
+## When will devs be able to deploy on mainnet? How can I create my first account for mainnet?
 
-During the beta period, you will need to submit your contract to the Flow team for review and then you’ll be able to submit it to the network.
+As soon as [permissionless deployment (ETA summer 2022)](https://permissionless.onflow.org/) is released, you can create accounts and deploy contracts to mainnet directly.
```

Comment:
Aren't accounts already possible to create prior to permissionless deployment? Can we stub these 2 questions into two separate ones for better SEO?

Also, let's keep the questions super simple: "How can I deploy a contract to mainnet?" or "How to deploy a contract to mainnet", That way it doesn't need to change when permissionless rolls out, and it's easier to find for devs